cmd/miqt-docker/main.go

```diff
@@ -151,13 +152,38 @@ func getDockerRunArgsForGlob(dockerfiles []fs.DirEntry, containerNameGlob string
 		fullCommand = append(fullCommand, "-t")
 	}
 
-	if needsSudo && runtime.GOOS != "windows" {
+	// On Linux, Docker runs as root, and our generated files would be owned by 0:0
+	// Use `--user $uid:$gid` to avoid this
+	// This is not needed for podman, and it's not needed for Windows, and it's
```

Comment:
it's not needed but is it harmful? why not always supply it by default and have an option to disable?